### ISSUE TITLE
Upgrade to React 17

### ISSUE THREAD

**Issue by PlanetIrata** [kevinsqi/react-circular-progressbar#167]
### Summary of issue

Can't `npm update` a project already upgraded to React 17.0.1

```bash
npm ERR! Found: react@17.0.1
npm ERR! node_modules/react
npm ERR!   react@"^17.0.1" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^0.14.0 || ^15.0.0 || ^16.0.0" from react-circular-progressbar@2.0.3
```

Workaround (bad): `npm update --force`

**Comment by Agillmann** [comment]
I have the same issue but with this command it working good :

**Comment by prince272** [comment]
I'm also facing the same issue here ooo.

**Comment by kevinsqi** [comment]
I released an update which allows ^17.0.0 in version 2.0.4. Thanks for reporting.
